go-controller/pkg/node/node_dpu.go

```diff
@@ -29,6 +30,9 @@ func (n *OvnNode) watchPodsDPU(isOvnUpEnabled bool) error {
 	podLister := corev1listers.NewPodLister(n.watchFactory.LocalPodInformer().GetIndexer())
 	kclient := n.Kube.(*kube.Kube)
 
+	// Support default network for now
```

Comment:
this is possible with current code for sriov case, right? so it is not unexpected.

Reply:
How is that possible? Up until now all attachments are to the default primary network, aren't they?
And in the comment there you mention only supporting attachments to the default network.
So I am suggesting if we could detect circumstances that don't match your comment and error out instead of making assumptions.

Reply:
sorry, my previous comments seems to answer a different question (about configureOVS() can be called for non-default-network, but not for dpu case).

but here in watchPodsDPU() function called by ovnkube-node, it is unware of any secondary networks, and it only handles the default network here), not sure what to detect. 

basically, if there are secondary networks makes use of dpu, it is silently ignored and cni will not be able to plumb the second interface for the pod (timeout when waiting for dpuStatus annotation)

Reply:
OK. In the CNI side we could then check for this an give a more meaningful error other than timeout?

Reply:
I will add dpu support for secondary networks in the later MR. do you still want me to fix this error issue which is temporary?